src/bracketing.jl

```diff
@@ -112,7 +119,28 @@ end
 
 ####
 ## find_zero interface.
+"""
+    `Bisection()`
```

Comment:
In markdown syntax, indenting by four spaces activates code-formatted blocks, the same as triple backticks. So if you use backticks inside of an indented line like this, they'll show up in the rendered markdown. That is, rather than appearing as

    Bisection()

this will appear as

    `Bisection()`

Thus I recommend removing the backticks around the function signatures here and elsewhere in docstrings.